Libplanet/Tx/TxMetadata.cs

```diff
@@ -81,24 +80,45 @@ public TxMetadata(Bencodex.Types.Dictionary dictionary)
             ).ToUniversalTime();
         }
 
-        /// <inheritdoc cref="ITxMetadata.Nonce"/>
+        /// <summary>
+        /// The number of previous <see cref="Transaction{T}"/>s committed by
+        /// the <see cref="Signer"/> of this transaction.  This nonce is used for preventing
+        /// replay attack.
+        /// </summary>
         public long Nonce { get; set; }
 
-        /// <inheritdoc cref="ITxMetadata.Signer"/>
+        /// <summary>
+        /// A <see cref="PublicKey"/> of the account who signs this transaction.
```

Comment:
Isn't it the `Address` of the account instead of `PublicKey`?

Reply:
Amended.